extensions/amp-story/1.0/progress-bar.js

```diff
@@ -218,6 +213,16 @@ export class ProgressBar {
     return root;
   }
 
+  /**
+   * Whether the given id corresponds to an ad page.
+   * @param {string} id
+   * @return {boolean}
+   * @private
+   */
+  isAdPage_(id) {
```

Comment:
Sorry about this, I'll keep being picky about how we name things, I believe this is super important for our ever growing team and writing easy to maintain code. We're talking about segments in this file, not pages

```suggestion
  isAdSegment_(id) {
```